subxt/src/config/polkadot.rs

```diff
@@ -4,15 +4,29 @@
 
 //! Polkadot specific configuration
 
+use super::{
+    extrinsic_params::{BaseExtrinsicParams, BaseExtrinsicParamsBuilder},
+    Config,
+};
 use codec::Encode;
 
-use super::extrinsic_params::{BaseExtrinsicParams, BaseExtrinsicParamsBuilder};
+pub use crate::utils::{AccountId32, MultiAddress, MultiSignature};
+use crate::SubstrateConfig;
+pub use primitive_types::{H256, U256};
 
 /// Default set of commonly used types by Polkadot nodes.
-pub type PolkadotConfig = super::WithExtrinsicParams<
-    super::SubstrateConfig,
-    PolkadotExtrinsicParams<super::SubstrateConfig>,
->;
+pub enum PolkadotConfig {}
+
+impl Config for PolkadotConfig {
```

Comment:
I do wonder, we implement the `Config` directly to gain more flexibility for the `PolkadotConfig`?

Reply:
Yeah, `WithExtrinsicParams` only lets you change the extrinsic params bit; Tadeo needed to change another param so implementing Config "properly" is the way to go :)